sdk/identity/Azure.Identity/README.md

```diff
@@ -1,99 +1,175 @@
 
-Use the guidelines in each section of this template to ensure consistency and readability of your README. The README resides in your package's GitHub repository at the root of its directory within the repo. It's also used as the package distribution page (NuGet, PyPi, npm, etc.) and as a Quickstart on docs.microsoft.com. See [README-EXAMPLE.md](README-EXAMPLE.md) for an example following this template.
+  [Source code][source] | [Package (nuget)][package] | API reference documentation (Coming Soon) | [Azure Active Directory documentation][aad_doc]
 
-**Title**: The H1 of your README should be in the format: `# [Product Name] client library for [Language]`
 
-* All headings, including the H1, should use **sentence-style capitalization**. Refer to the [Microsoft Style Guide][style-guide-msft] and [Microsoft Cloud Style Guide][style-guide-cloud] for more information.
-* Example: `# Azure Batch client library for Python`
 
-**Introduction**: The introduction appears directly under the title (H1) of your README.
+## Getting started
 
-* **DO NOT** use an "Introduction" or "Overview" heading (H2) for this section.
-* First sentence: **Describe the service** briefly. You can usually use the first line of the service's docs landing page for this (Example: [Cosmos DB docs landing page](https://docs.microsoft.com/azure/cosmos-db/)).
-* Next, add a **bulleted list** of the **most common tasks** supported by the package or library, prefaced with "Use the client library for [Product Name] to:". Then, provide code snippets for these tasks in the [Examples](#examples) section later in the document. Keep the task list short but include those tasks most developers need to perform with your package.
-* Include this single line of links targeting your product's content at the bottom of the introduction, making any adjustments as necessary (for example, NuGet instead of PyPi):
+### Install the package
 
-  [Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/azure-batch) | [Package (PyPi)](https://pypi.org/project/azure-batch/) | [API reference documentation](https://docs.microsoft.com/python/api/overview/azure/batch?view=azure-python) | [Product documentation](https://docs.microsoft.com/azure/batch/)
+Install the Azure Identity client library for .NET with [NuGet][nuget]:
 
-> TIP: Your README should be as **brief** as possible but **no more brief** than necessary to get a developer new to Azure, the service, or the package up and running quickly. Keep it brief, but include everything a developer needs to make their first API call successfully.
+```PowerShell
+Install-Package Azure.Identity -IncludePrerelease
+```
 
-## Getting started
+### Prerequisites
+* An [Azure subscription][azure_sub].
+* An existing Azure Active Directory service principal. If you need to create a service principal, you can use the Azure Portal or [Azure CLI][azure_cli].
```

Comment:
Is this actually a pre-requisite? I see that we use it for some of the examples, but if we go with the certificate route, we won't need this. right?

Reply:
No, you'd still need a service principal. Think of the certificate as a fancy secret.

The details of a service principal known to the user aren't required for managed identity though. I'm uncertain that's worth calling out here. It would further muddy the water for newbies.

Reply:
Ohh nice. I had no idea. Thanks !